public/app/features/dashboard/components/DashNav/DashNav.tsx

```diff
@@ -67,7 +67,7 @@ class DashNav extends PureComponent<Props> {
 
   onClose = () => {
     this.props.updateLocation({
-      query: { edit: null, viewPanel: null },
+      query: { editPanel: null, viewPanel: null },
```

Comment:
This should only be viewPanel: null , but it does not actually matter

Reply:
Ok ok, super! Will remove the `editPanel:null`